packages/block-library/src/calendar/index.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Server-side rendering of the `core/calendar` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/calendar` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the block content.
+ */
+function render_block_core_calendar( $attributes ) {
+	global $monthnum, $year, $post;
+	$previous_monthnum;
+	$previous_year;
+
+	// phpcs:disable WordPress.WP.GlobalVariablesOverride.OverrideProhibited
```

Comment:
nit: I'd prefer that we only disable this rule for the specific lines that need it disabled. That way, when a future developer adds more code here, it's still being properly linted. This can be done using `phpcs:ignore`:

```php
if ( isset( $attributes['month'] ) ) {
	$previous_monthnum = $monthnum;
	// phpcs:ignore WordPress.WP.GlobalVariablesOverride.OverrideProhibited
	$monthnum          = $attributes['month'];
}

if ( isset( $attributes['year'] ) ) {
	$previous_year = $year;
	// phpcs:ignore WordPress.WP.GlobalVariablesOverride.OverrideProhibited
	$year          = $attributes['year'];
}
```

https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file